src-java/floodlight-service/floodlight-modules/README.md

```diff
@@ -15,3 +15,45 @@ where:
  ratePkts - is the rate in packets for specific type of rule
  disco_packet_size - the size of discovery packet, is being defined in properties file with the key 'org.openkilda.floodlight.switchmanager.SwitchManager.disco-packet-size'.
 ```
+
+#### How to debug Floodlight component
+
+In order to run floodLight in the debug mode we have to add this line of arguments
+```
+"-agentlib:jdwp=transport=dt_socket,address=50506,suspend=n,server=y”
+``` 
+when running the app.
+Particularly in floodLight we can not do this straight to the Dockerfile since it leads us to entrypoint.sh script.
+We have to add these arguments to the
+```
+projectroot/docker/floodlight-modules/entrypoint.sh
+```
+file;
+
+
+in this file find `exec java` line under the ```"$1" = 'floodlight’```  condition.
+In the end it should look something like this:
+```
+exec java "-agentlib:jdwp=transport=dt_socket,address=50506,suspend=n,server=y" -XX:+PrintFlagsFinal……… the rest arguments….;
+```
+then we have to expose debugging port in docker-compose.tmpl file.
+go to ```projectroot/confd/templates/docker-compose/docker-compose.tmpl```
+find floodlight_1 and add desirable port mapping:
+```ex: 50506:323232     - (inside:outside).```
+In this example, 50506 is the port number that we have pointed for entrypoint.sh file previously,
```

Comment:
`for entrypoint.sh` doesn't sound correct to me. I'd say: ... that we used in entrypoint.sh ...

Reply:
fixed, thank you